### DISCUSSION TITLE
Move Search to fixed header rather than scrolling off the top

### DISCUSSION THREAD

**Discussion by jaredzimmerman** [Authenticator-Extension/Authenticator#1119] [Ideas]
If i start to scroll, then decide to search for a site, it would be much easier to have the search fixed to the top of the UI chrome rather than scrolling off like its a site code

**Comment by Sneezry** [comment]
You can press "/" as a shortcut.

**Comment by jaredzimmerman** [comment]
sure, but just fixing the search to the top seems much more accessible and obvious
